src/main/scala/is/hail/variant/VariantDataset.scala

```diff
@@ -41,11 +41,6 @@ object VariantDataset {
 
 class VariantDatasetFunctions(private val vds: VariantDataset) extends AnyVal {
 
-  private def requireSplit(methodName: String) {
-    if (!vds.wasSplit)
-      fatal(s"method `$methodName' requires a split dataset. Use `split_multi' or `filter_multi' first.")
-  }
-
```

Comment:
you don't think this error message is helpful enough to our users to justify this method's existence?